docs/topics/cli-run.adoc

```diff
@@ -52,7 +54,9 @@ The following command analyzes the `org.jboss.seam` packages of the link:https:/
 
 [source,terminal,subs="attributes+"]
 ----
-$ <{ProductShortName}_HOME>/{mta-cli} --sourceMode --input /path/to/seam-booking-5.2/ \
+
+$ <{ProductShortName}_HOME>/bin/windup-cli --sourceMode --input /path/to/seam-booking-5.2/ \
```

Comment:
The `bin` folder should be removed.

The `--sourceMode` option no longer exists, it is `--mode source-only` now

Should we reference `windup-cli` directly instead using a placeholder `{mta-cli}` like we were doing?

Reply:
I will push changes to MTR when we have a bit more time

Eli has deleted the `windup-cli` attribute